addons/crm/models/crm_team_member.py

```diff
@@ -58,6 +58,19 @@ def _get_lead_month_domain(self):
             ('date_open', '>=', limit_date),
         ]
 
+    # ------------------------------------------------------------
+    # CRUD
+    # ------------------------------------------------------------
+
+    @api.model_create_multi
+    def create(self, vals_list):
+        """ Override to remove autofollow. No need to follow team members
+        when creating them as chatter is mainly used for information purpose
+        (tracked fields). """
+        return super(TeamMember, self.with_context(
+            mail_create_nosubscribe=True
+        )).create(vals_list)
+
```

Comment:
Isn't it better to add this on the act_window (`sales_team.crm_team_member_action`)?
```xml
<field name="context">{'mail_create_nosubscribe': True}</field>
```

Reply:
Well I prefer to keep it at model level. As this is quite a technical record, default behavior should always be to avoid automatic followers ... unless you write or ask it :) . So I think creating a member should never make you follower of it.

Reply:
ah ok, even if it's not created in the view I see :)

Reply:
shouldn't you make in in the sale_team module ?

Reply:
That is ... not untrue ^^ forgot that sales_team depends on mail and already adds the chatter. Done Sir.